src/routes/delegates/entities/__tests__/delete-delegate-dto.builder.ts

```diff
@@ -0,0 +1,10 @@
+import { faker } from '@faker-js/faker';
+import { Builder, IBuilder } from '../../../../__tests__/builder';
+import { CreateDelegateDto } from '../create-delegate.entity';
+
+export function deleteDelegateDtoBuilder(): IBuilder<CreateDelegateDto> {
```

Comment:
Shouldn't this be a builder for `DeleteDelegateDto`?

Reply:
Of course! This was a mistake (properties of both classes overlap so the compiler didn't complaint 😄). I will fix it!

Reply:
Fixed in [74dde61](https://github.com/5afe/safe-client-gateway-nest/pull/270/commits/74dde610ba8bbbdcb48c03380f2c23a1a6c9f9fc)